_dashboards/management/multi-data-sources.md

```diff
@@ -141,9 +143,65 @@ data_source.hideLocalCluster: true
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboards/multidata-hide-localcluster.gif" alt="Multiple data sources hide local cluster">{: .img-fluid}
 
+## Using multiple data sources with external dashboards plugins
+Introduced 2.14
+{: .label .label-purple }
+
+The following plugins now support multiple data sources
+
+### Index management
+
+When the data source feature is enabled, you can navigate to **Index Management** under the **Management** menu. Using indexes as an example, you can view all connected data sources and select a specific one from the navigation bar at the upper right. By default, the indexes from the designated default data source is selected are displayed. However, you can select any connected data source to view its corresponding indexes. The following GIF illustrates these steps.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ism_mds1.gif" alt="Multiple data sources in ISM list page"/>
+
+To perform operations on a specific index within a data source, select the individual index from the list. To create a new index, select the `Create Index` button, which opens a form. Fill in the required information and select the `Create` button. The index is created within the selected data source. The following GIF illustrates these steps. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ism_mds2.gif" alt="Multiple data sources in ISM create page"/>
+
+### Anomaly detection
+
+When the data source feature is enabled, you can navigate to **Anomaly Detection** under the **OpenSearch Plugins** menu. In the navigation bar at the upper right, you can view all connected data sources and select a specific data source to view the dashboard from that source if the data source has detectors. If the selected data source does not have any detectors, the page prompts you to `Create detector`. The following GIF illustrates these steps.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ad_mds1.gif" alt="Multiple data sources in Anomaly Detection dashboard page"/>
+
+When you select **Detectors** from the side bar, the page displays the detectors currently configured for the currently selected data source. You can view and configure individual detectors by selecting them from the list. These steps are shown in the following GIF.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/ad_mds2.gif" alt="Multiple data sources in Anomaly Detection detector page"/>
+
+### Security
+
+When the data source feature is enabled, you can navigate to **Security** under the **Management** menu. Using roles management as an example, you can view all connected data sources in the navigation bar at the upper right and select a specific data source to view its existing roles. To create a new role, select the **Create role** button, which takes to a new create page. Fill in the required information and select **Create** to add the new role to the selected data source. The following GIF illustrates these steps.
```

Comment:
The end of the third sentence needs some correction.

Reply:
Could you clarify?

Reply:
I suspect instead of "which takes to a new create page", we mean "which takes you to a new page."